quipucords/quipucords/settings.py

```diff
@@ -25,8 +25,8 @@
 
 env = environ.Env()
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = Path(__file__).absolute().parent.parent
+DEFAULT_DATA_DIR = BASE_DIR.parent / "var"
```

Comment:
quipucords/var is already part of this repo as a "garbage" place. It has the convenience of the original implementation (fewer variables to worry about) with defaults that will persist (since tmp will be cleaned on restart), but no garbage in the middle of source code.

I like the idea of putting it under a env var, tho. How about I just make it configurable? I must add, tho that with current implementation `DJANGO_SECRET_KEY`, `DJANGO_DB_PATH` and `DJANGO_LOG_FILE` will override this config - which I think is the way to go. The new `QPC_LOG_DIRECTORY` will also override the default dir.

Reply:
Eh, if this is mostly just preserving the existing behavior, then I don't think we need more variables. Thanks for explaining, though. Reading the many `parent`s, I thought this was going outside the quipucords project directory.

Reply:
> Reading the many parents, I thought this was going outside the quipucords project directory.

yeah, it can be confusing haha. to make this clear, I'll add a comment pointing `BASE_DIR` is `quipucords/quipucords` and `DEFAULT_DATA_DIR` is `quipucords/var`